modules/ROOT/pages/endpoint-as2-send.adoc

```diff
@@ -7,44 +7,57 @@ Create the AS2 identifiers for your organization and your trading partners in th
 [%header%autowidth.spread]
 |===
 |Setting |Description |Required
+
+| Owner
+| The trading partner or host that owns the endpoint. The value in this field is populated automatically and cannot be modified.
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window.
+| Yes
+
+| Partner
+|
+| Yes
```

Comment:
We need to remove this row with the setting "Partner"

Reply:
Done.